### DISCUSSION TITLE
Is there a way to open a Project (Beta) under an organization or repository?

### DISCUSSION THREAD

**Discussion by dolev-isp** [community/community#6471] [Projects and Issues]
Hi, 
We'd like to try out the new Projects (Beta), but thus far I only managed to create a personal Project. Can we do it in the organization or repository level so anyone in the team can use it? Better yet, can we convert an existing project, or at least re-use its issues?

**Comment by stafyniaksacha** [comment]
Hi! You should opt in for each organization on the feature page here: https://github.com/features/issues

**Comment by clapfootmark** [comment]
I waited for months to get into the beta only to just find out now in this thread that I couldn't use it for my organization. I feel that if you got in personally, you should be able to quickly enroll your organization into the beta as well instead of having to wait for another indefinite period of time.

**Reply by stafyniaksacha** [reply]
You can go here to ping Github: https://github.com/github/feedback/discussions/5023#discussioncomment-1458597
